docs/ABOUT.md

```diff
@@ -38,6 +38,17 @@ Another common implementation is `mawk`, which is based on a bytecode interprete
 This track will use `gawk`, GNU's awk implementation.
 `gawk` may not have the raw speed of `mawk`, but it is very feature-packed, well documented, and has some handy command-line options.
 
+### shebang
+
+Depending on the platform, awk scripts can be executed by adding a "shebang" line to the top of the file.
+One most systems, one of the following two shebangs ought to work.
+However, since there is no one shebang which works for all systems, we have choosen to omit shebangs.
+
+```bash
+#!/usr/bin/env -S gawk -f
+#!/usr/bin/gawk -f
+```
+
```

Comment:
Even an invalid shebang like `#!gawk` can act a hint to your editor to apply the proper syntax highlighting.